fe/src/pages/profile.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div
+    class="py-6 px-36 md:px-44 min-h-screen lg:px-48 bg-linear-gradient-white-to-light overflow-auto"
+  >
+    <base-user-card
+      :block-chain-value="blockChainValue"
+      :block-chain-label="blockChainLabel"
+      :img-src="imgSrc"
+      :user-id="userId"
+    />
+    <div class="mt-6">
+      <base-tabs
+        :active-tab="activeTab"
+        :items="profileTabs"
+        @change-tab="activeTab = $event"
+      />
+    </div>
+    <div v-if="activeTab === 'My Campaigns'">
+      <div
+        class="mt-6 grid grid-cols-3 gap-6 sm:grid-cols-1 lg:grid-cols-2 xl:grid-cols-3 campaigns"
+      >
+        <base-campaign-card
+          v-for="i in 6"
```

Comment:
Improvement:
Noticed the use of magic numbers such as this. 
```suggestion
          v-for="item in items"
```
Instead, create a sample json file in mocks folder for instances like this. To simulate actual data an for easier integration in the future.